code/game/machinery/fax_machine.dm

```diff
@@ -515,6 +515,85 @@ GLOBAL_LIST_EMPTY(alldepartments)
 	target_department = "Brig"
 	network = "USCM High Command Quantum Relay"
 
+/obj/structure/machinery/faxmachine/backpack
+	name = "\improper Portable Press Fax Machine"
+	desc = "A standard issue portable fax machine for civilian reporters. Functions off of an internal battery. Cannot receive faxes while being worn. It is currently deployed. Click-drag the device towards you to pick it up."
+	needs_power = FALSE
+	use_power = USE_POWER_NONE
+	icon_state = "fax_backpack"
+	health = 150
+
+/obj/item/device/fax_backpack
+	name = "\improper Portable Press Fax Machine"
+	desc = "A standard issue portable fax machine for civilian reporters. Functions off of an internal battery. Cannot receive faxes while being worn. It is currently undeployed. Activate the device inhand to deploy it."
+	w_class = SIZE_HUGE
+	flags_equip_slot = SLOT_BACK
+	flags_item = ITEM_OVERRIDE_NORTHFACE
+	icon = 'icons/obj/structures/machinery/library.dmi'
+	icon_state = "fax_backpack"
+	item_state = "fax_backpack"
+
+/obj/item/device/fax_backpack/attack_self(mob/user) //activate item version fax inhand to deploy
+	..()
+	if(!ishuman(user))
+		return
+	var/turf/T = get_turf(usr)
+	if(istype(T, /turf/open))
```

Comment:
Additionally, do not use single letter vars. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations

Please refactor those to meet code style.